pkgs/development/interpreters/erlang/R22.nix

```diff
@@ -0,0 +1,6 @@
+{ mkDerivation }:
+
+mkDerivation rec {
+  version = "22.0.3";
+  sha256 = "1z5f92m8lks6j48fbfb4mp7wzprr9bv842xqp54az2xb8g2dcfnh";
```

Comment:
darwin builds needs the same workaround for the sw_vers impurity as R21.

Reply:
Thanks. Should be fixed now.